content/account/settings/index.textile

```diff
@@ -0,0 +1,74 @@
+
+h2. Account menu overview
+
+Your account menu provides the following features to manage all aspects of your account:
+
+<a href="@content/screenshots/dash/account.png" target="_blank">
+  <img src="@content/screenshots/dash/account.png" style="width: 140%" alt="Ably Account Settings">
+</a>
+
+h3. Settings
+
+Manage your Ably account "settings"::/account/setting/settings, including authentication, billing, and ownership transfer options.
+
+* Use your account's unique identifier (Account ID) for managing integrations or when contacting support. Additionally, account owners can require all users to set up two-factor authentication (2FA) for added security.
+* Account owners can update billing and package details and are the only ones who can modify these settings or transfer ownership of the account. To transfer ownership or cancel the account, contact Ably support.
+
+h3. Billing
+
+Manage your "billing":/pricing/billing and package settings to scale your services:
+
+* Review your current package to assess if it fits your needs, whether free, pay-as-you-go, or committed use.
+* Ensure your billing information is up to date, including company name, billing email (optional), address, international phone number, and timezone.
+* View and manage invoices and payment methods once your account has been upgraded.
+
+h3. Limits
+
+Understand and manage your package "limits":/pricing/limits by tracking usage across messages, data, connections, and channels:
+
+* Limits are applied to prevent service disruption and vary by package.
+* Limits can be time-based, quantity-based, or rate-based, and they may apply either per connection/channel or across the entire account.
+* Some limits allow temporary overages without immediate restriction—prompting you to upgrade your package — while other limits are strictly enforced to prevent service disruption when exceeded.
+* Notifications are provided when limits are nearing or exceeded, and detailed logs of warnings and exceeded limits are accessible in the account dashboard.
+
+h3. Usage
+
+Monitor your account's resource consumption with detailed usage statistics:
+
+* The usage statistics table monitors app performance by tracking resource usage, such as messages, data transferred, connections, and channels.
+* The statistics chart visualizes app usage data over time, allowing users to define specific time ranges, zoom in on different periods, and analyze metrics in detail.
+
+h3. Users
+
+Manage the "users":/account/settings/users associated with your account:
+
+* The account owner role has full permissions to manage the account, including inviting and removing users, and assigning roles like *developer*, *billing*, or *admin*.
+* Multiple roles can be assigned to a single user.
+* Remove, or change user roles, as well as how to delete an Ably account.
```

Comment:
We have a section called "delete your account" at the bottom. Which to be fair -- directs them to a [FAQ](https://faqs.ably.com/how-do-i-cancel-my-account). I can remove this part of the bullet point, to avoid confusion though :)